### PR TITLE
version bump to 0.5.9

### PR DESCRIPTION
**Pull request by stephanielee9**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -247,10 +247,10 @@ jobs:
         include:
           - type: "qiskit-superstaq"
             pattern: "*_qss.ipynb qiskit-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```
Comment (richrines1) [inline .github/workflows/ci.yml:250]: nit
```suggestion
            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq -e ./supermarq-benchmarks"
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -247,10 +247,10 @@ jobs:
...
           - type: "cirq-superstaq"
             pattern: "*_css.ipynb cirq-superstaq"
-            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```
Comment (richrines1) [inline .github/workflows/ci.yml:253]: this one shouldn't be necessary - `qiskit-superstaq` is already included in `cirq-superstaq[examples]` (we need it for ibm pulse examples)
```suggestion
            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```

**Review by richrines1 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (richrines1) [reply .github/workflows/ci.yml]: ahh sorry this was wrong :/ we still need it to make sure we install the local version

Comment (bharat-thotakura) [reply .github/workflows/ci.yml]: Sounds good, fixed in [17a46fd](https://github.com/Infleqtion/client-superstaq/pull/899/commits/17a46fd37ef5a3916ed063147aa2a0280b2606b5).

**Review by richrines1 — APPROVED** [review]
lgtm!

**Pull request merged**